src/main/java/com/google/gerrit/extensions/client/ListChangesOptionByVersion.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2012 The Android Open Source Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.gerrit.extensions.client;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ListChangesOptionByVersion {
+  static final Map<String, ListChangesOption> maxChangeOptionByVersion =
+    Stream.of(new Object[][]{
```

Comment:
Can you make this type-safe? Or probably even better init the map with a `static` block (https://stackoverflow.com/a/507658)?

Reply:
Done.